### DISCUSSION TITLE
Add "Close Window" Button For Mouse Only Use

### DISCUSSION THREAD

**Discussion by Gavski** [f1multiviewer/issue-tracker#459] [Feature requests] [answered]
For a TV connected computer, in this case a Mac, to close a window requires a "Ctrl + Click".
This is the only time the keyboard is needed to use the app in a linear flow/domestic environment and it's pretty annoying!
It would be great to add a mousable button in the window to close a window.  
Eg an "X" in the top corner.
Then, I only need to keep my trackpad to hand, the keyboard can stay at a distance for only occasional use.
Thanks

**Comment by f1multiviewer** [comment, marked as answer]
There's two ways you can do this right now:

1. Bring your mouse close to the top-left corner, it'll show the stoplight
2. Enable two-finger click to open context menu to close

**Comment by Gavski** [comment]
Excellent - would never have found this.
Thx
Jon
